person.rb

```diff
@@ -0,0 +1,31 @@
+require './nameable'
+
+class Person < Nameable
+  def initialize(age, name = 'Unknown', _parent_permission: true)
+    super()
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @_parent_permission = parent_permission
+    @visits =  []
+  end
```

Comment:
Great job @Therealadetunji 🤝

- [ ] Kindly initialize @rentals. Please check this [link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/class_and_object_for_irb.md#constructor-and-instance-vars) for more guidance.
- [ ] You may also add a method for adding rentals. Kindly check this [link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) for more guidance.